kANe/v4/2_antyeShTi_Ashaucha_shuddhi/07_Eschatology.md

```diff
@@ -336,53 +336,40 @@ H. D. 27
 
- _ ___494, अधानमिचितः। एतदेव भूनिजोपणमेतत्सनान कर्न यदन्यदग्निकर्मण.। Taquar. XIII 8,4,11
+[^494]: अधानग्निचितः। एतदेव भूमिजोपणमेतत्सनान कर्म यदन्यदग्निकर्मण.। शतपथवा . XIII 8,4,11
```

Comment:
plz send image for this - doubtful

Reply:
![image](https://user-images.githubusercontent.com/101622645/164916494-c022f975-d3d9-4d0a-9b80-823b1dce02b0.png)

Reply:
I see several mistakes. First word - अथानग्निचितः। 

Please use new pdf I sent - much clearer. Please send image from there - I'll show a trick by which you can make correction much easier.

Reply:
![image](https://user-images.githubusercontent.com/101622645/164934206-67e88b18-d2d6-4429-9937-e7998364b880.png)
This is from new PDF.

Reply:
No - this is not the new pdf. The below is from the new pdf. See the difference? Can you make sure you got the correct pdf?

![image](https://user-images.githubusercontent.com/2664797/164951470-af4b1822-5f97-4b07-ab96-0f9246accd44.png)